spec/index.bs

```diff
@@ -13951,23 +13966,10 @@ dictionary GPUCanvasConfiguration {
 ### Canvas Color Space ### {#canvas-color-space}
 
 During presentation, the color values in the canvas are converted to the color
-space of the screen. Color values are then clamped to the `[0, 1]` interval in
-the color space of the screen.
-
-<div class=note heading>
-    For example, suppose that the value `(1.035, -0.175, -0.140)` is written to an
-    `'srgb'` canvas.
+space of the screen.
 
-    If this is presented to an sRGB screen, then this will be converted to sRGB
-    (which is a no-op, because the canvas is sRGB), and then will be clamped to
-    the sRGB value `(1.0, 0.0, 0.0)`.
-
-    If this is presented to a Display P3 screen, then this will be converted to
-    the value `(0.948, 0.106, 0.01)` in the Display P3 color space, and no
-    clamping will be needed.
-</div>
```

Comment:
(Merge conflict with #4499 - that example was superseded by the other examples added in this PR so I removed it.)